lib/matplotlib/container.py

```diff
@@ -18,7 +18,7 @@ def __repr__(self):
     def __new__(cls, *kl, **kwargs):
         return tuple.__new__(cls, kl[0])
 
-    def __init__(self, kl, label=None):
+    def __init__(self, kl, label=None, lorder=''):
```

Comment:
Python 3 doesn't accept a mixture of ints and strings when sorting, so the empty string is a non-starter.